contribs/gmf/less/mobile.less

```diff
@@ -253,3 +253,20 @@ button[ngeo-mobile-geolocation] {
     }
   }
 }
+
+//mobile styles for the theme and background selector
```

Comment:
Please change this comment to: `// Mobile styles for the theme and background selector.`
(Space first, then an uppercase and a point at the end).